source/MonthlyOverview/MonthlyStyles.css

```diff
@@ -53,11 +52,6 @@ body {
 
 #index > a {
     font-size: 3.5vh;
-    text-decoration: none;
-}
```

Comment:
I feel like keeping this decoration would be good to have to help make it clear to users that this is clickable